chapters.md

```diff
@@ -17,7 +17,62 @@ As an example, consider a `Parent ChapterAtom Element` that has its `ChapterFlag
 
 Three `Edition Flags` are defined to describe the bevahior of the `EditionEntry Element`: `EditionFlagHidden`, `EditionFlagDefault` and `EditionFlagOrdered`.
 
-The `EditionFlagHidden` Flag behaves similarly to the `ChapterFlagHidden` Flag: if `EditionFlagHidden` is set to `true` then its `Child ChapterAtoms Elements` MUST also be interpretted as if their `ChapterFlagHidden` is also set to `true`, regardless of their own `ChapterFlagHidden` flags. If the `EditionFlagHidden` is toggled by a `Control Track` to `false` then the `ChapterFlagHidden` Flags of the `Child ChapterAtoms Elements` SHALL determine if the `ChapterAtom` is hidden or not.
+
+#### `EditionFlagHidden`
+
+The `EditionFlagHidden Flag` behaves similarly to the `ChapterFlagHidden Flag`: if `EditionFlagHidden` is set to `true`, its `Child ChapterAtoms Elements` MUST also be interpreted as if their `ChapterFlagHidden` is also set to `true`, regardless of their own `ChapterFlagHidden Flags`. If `EditionFlagHidden` is toggled by a `Control Track` to `false`, the `ChapterFlagHidden Flags` of the `Child ChapterAtoms Elements` SHALL determine whether the `ChapterAtom` is hidden or not.
+
+
+#### `EditionFlagDefault`
+
+It is recommended that only one Edition's `EditionFlagDefault Flag` is set to `true`. If more then one `Default Edition` is present, the first `Default Edition` MUST be used.
+If an `EditionFlagDefault Flag` is set to `true`, this `Edition` MUST be used. When all `EditionFlagDefault Flags` are set to `false`, the first Edition MUST be used.
+
+
+#### `EditionFlagOrdered`
+
+The `EditionFlagOrdered Flag` is a significant feature.
+
+If the `EditionFlagOrdered Flag` is set to `false`, "Matroska Simple-Chapters" are used and only the `ChapterTimeStart` of a chapter is used as chapter mark to jump to the predefined point in the time line.
+Many chapter elements like `ChapterTimeEnd` or `ChapterSegmentUID` SHOULD be now ignored by the playback application.
```

Comment:
Removing elements just to introduce others that do the same doesn't make sense. If this stuff is used, then we cannot and should not deprecate or even remove them. Simple as that.